packages/elastic-apm-synthtrace/src/scripts/run_synthtrace.ts

```diff
@@ -213,4 +218,5 @@ yargs(process.argv.slice(2))
       }
     }
   )
+  .showHelpOnFail(false)
```

Comment:
what is the point of this?

Reply:
Will remove it, since we execute inside the command handler any exception will cause the full help to be printed which is slightly annoying. I could not find a nicer way to deal with this by glancing over the yargs docs.